Sources/scipio/Options.swift

```diff
@@ -38,6 +38,11 @@ struct BuildOptionGroup: ParsableArguments {
           help: "Whether to enable Library Evolution feature or not")
     var shouldEnableLibraryEvolution = false
 
+    @Flag(name: [.customLong("--strip-dwarf-symbols")],
+          inversion: .prefixedNo,
+          help: "Whether to strip DWARF symbol from built binary or not")
+    var shouldStripDWARFSymbols: Bool = false
```

Comment:
We should describe that the option is only for static frameworks (both on its name and help).